adapter/service/outlvlswitch/routing.go

```diff
@@ -168,3 +242,29 @@ func HandleCmdLvlGetReport(serviceRegistry adapter.ServiceRegistry) router.Messa
 		}),
 	)
 }
+
+func getStartLvl(message *fimpgo.Message) (int, error) {
+	switch d, ok, err := message.Payload.Properties.GetIntValue(StartLvl); {
+	case !ok:
+		log.Info("adapter: start_lvl not found in message properties")
+
+		return 0, nil
+	case err != nil:
+		return 0, err
+	default:
+		return int(d), nil
+	}
+}
+
+func getDurationInSeconds(message *fimpgo.Message) (time.Duration, error) {
+	switch d, ok, err := message.Payload.Properties.GetIntValue(Duration); {
+	case !ok:
+		log.Info("adapter: duration not found in message properties")
```

Comment:
Do not log if this optional property is not provided. It's normal case, it does not require logging.